exactonline/http.py

```diff
@@ -25,9 +25,9 @@
 except ImportError:  # python2
     import urllib2 as request
 try:
-    from urllib.parse import urljoin
+    from urllib.parse import urljoin, quote
 except ImportError:  # python2
-    from urlparse import urljoin
+    from urlparse import urljoin, quote
```

Comment:
Thanks. Didn't catch that yet.

The above is wrong though.

Using this instead:
```
 try:
-    from urllib.parse import urljoin
+    from urllib.parse import urljoin, quote
 except ImportError:  # python2
+    from urllib import quote
     from urlparse import urljoin
```